ssa.go

```diff
@@ -594,14 +600,20 @@ func newSSAStyleFromString(content string, format map[int]string) (s *ssaStyle,
 				s.scaleX = astiptr.Float(f)
 			case ssaStyleFormatNameScaleY:
 				s.scaleY = astiptr.Float(f)
+			case ssaStyleFormatNameOutline:
+				s.outline = astiptr.Float(f)
+			case ssaStyleFormatNameShadow:
+				s.shadow = astiptr.Float(f)
+			case ssaStyleFormatNameSpacing:
+				s.spacing = astiptr.Float(f)
 			}
 		// Int
 		case ssaStyleFormatNameAlignment, ssaStyleFormatNameBorderStyle, ssaStyleFormatNameEncoding,
-			ssaStyleFormatNameMarginL, ssaStyleFormatNameMarginR, ssaStyleFormatNameMarginV,
-			ssaStyleFormatNameOutline, ssaStyleFormatNameShadow, ssaStyleFormatNameSpacing:
+			ssaStyleFormatNameMarginL, ssaStyleFormatNameMarginR, ssaStyleFormatNameMarginV:
 			// Parse int
 			var i int
 			if i, err = strconv.Atoi(item); err != nil {
+				fmt.Printf("%q\n", attr)
```

Comment:
Can you remove this `Printf`?

Reply:
Yes. Sorry, I was mean to remove it, but forgot.